kong/plugins/statsd/log.lua

```diff
@@ -274,6 +278,47 @@ if ngx.config.ngx_lua_version >= 10011 then
       end
     end
   end
+  metrics.lmdb_usage = function (_, message, metric_config, logger, conf, tags)
+    -- we don't need this for every request, send every 1 minute
+    -- also only one worker needs to send this because it's shared
+    if worker_id ~= 0 then
+      return
+    end
+    if kong.configuration.database ~= "off" then
+      return
+    end
+
+    local now = ngx_time()
+    if lmdb_metrics_last_sent + LMDB_METRICS_SEND_THRESHOLD < now then
+      lmdb_metrics_last_sent = now
+      local lmdb_info, err = lmdb_get_env_info()
+      if err then
+        kong.log.err("failed to get lmdb info: ", err)
+        return
+      end
+      if conf.tag_style then
+        local tags = {
+          ["node"] = hostname,
+        }
+        logger:send_statsd("lmdb.used_space",
```

Comment:
This name seems to be misleading. `lmdb.allocated_space` maybe?

Reply:
offline chat whit datong for this, I will continue to contribute to this change.